build.gradle

```diff
@@ -27,7 +27,7 @@ ext {
     automatticTracksVersion = '5.1.0'
     gutenbergMobileVersion = 'v1.121.0-alpha1'
     wordPressAztecVersion = 'v2.1.3'
-    wordPressFluxCVersion = 'trunk-c670e916346bdaa2379936dd8dbb6750766ca319'
+    wordPressFluxCVersion = 'trunk-be56f1479e1a74e6733fcceb1d7e97b39b51157f'
```

Comment:
I believe the reference to `be56f1479e1a74e6733fcceb1d7e97b39b51157f` fails because it does not exist on `trunk`, it existed in the now-deleted `issue/privateatomiccookie-numberformatexception-fluxchash` branch. I.e., the first part of `trunk-be56f1479e1a74e6733fcceb1d7e97b39b51157f` is inaccurate. 

I believe this was correct when it was set to https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/2d90f9a8abbb862ebc1689553006d76e44a0d3c8. The CI tasks failed earlier because the [publish FluxC task](https://buildkite.com/automattic/wordpress-fluxc-android/builds/3686#0190a2f1-9b59-402c-bc24-30e78dd38a69) was not finished prior to pushing the reference update. 

I believe if you push up a reference to https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/2d90f9a8abbb862ebc1689553006d76e44a0d3c8 the CI checks should now succeed.

In general, we either need to wait for the publish FluxC task finish successfully before pushing the reference change here or retry the failed CI tasks once the FluxC publish succeeds.

Reply:
> In general, we either need to wait for the publish FluxC task finish successfully before pushing the reference change here or retry the failed CI tasks once the FluxC publish succeeds.

Thanks, @dcalhoun - that was a silly mistake on my part, perhaps due to using a local FluxC build. At any rate, I've fixed this in [4891ebf](https://github.com/wordpress-mobile/WordPress-Android/pull/21049/commits/4891ebf9685e50bec11eea5d1f572dcaa81981cf)